src/antlr4/Cypher.g4

```diff
@@ -23,7 +23,8 @@ kU_CopyNPY
     : COPY SP oC_SchemaName SP FROM SP '(' SP? StringLiteral ( SP? ',' SP? StringLiteral )* ')' SP BY SP COLUMN ;
 
 kU_Call
-    : CALL SP oC_SymbolicName SP? '=' SP? oC_Literal ;
+    : CALL SP ( ( oC_SymbolicName SP? '=' SP? oC_Literal )
```

Comment:
open an issue about having StandaloneCall and InQueryCall/ StandaloneCall is a statement, and inQueryCall is a ReadingClause